core/src/main/scala/com/microsoft/azure/synapse/ml/nn/BallTree.scala

```diff
@@ -3,7 +3,6 @@
 
 package com.microsoft.azure.synapse.ml.nn
 
-import breeze.linalg.functions.euclideanDistance
```

Comment:
nit: No other changes in this file. Was this moved to a different path in breeze?

Reply:
`euclideanDistance` package structure is changed and the below line covers the import for it. so removed this line, as it is not required anymore.